operators/pkg/forge/services_test.go

```diff
@@ -129,17 +129,32 @@ var _ = Describe("Services forging", func() {
 				},
 				Expected: []corev1.ServicePort{
 					{Name: forge.GUIPortName, Protocol: corev1.ProtocolTCP, Port: forge.GUIPortNumber, TargetPort: intstr.FromInt(forge.GUIPortNumber)},
+					{Name: forge.MetricsPortName, Protocol: corev1.ProtocolTCP, Port: forge.MetricsPortNumber, TargetPort: intstr.FromInt(forge.MetricsPortNumber)},
 					{Name: forge.MyDrivePortName, Protocol: corev1.ProtocolTCP, Port: forge.MyDrivePortNumber, TargetPort: intstr.FromInt(forge.MyDrivePortNumber)},
 				},
 			}),
+			Entry("When the Environment is a Container", ServiceSpecCase{
+				Mutator: func(env *clv1alpha2.Environment) *clv1alpha2.Environment {
+					env.EnvironmentType = clv1alpha2.ClassContainer
+					env.GuiEnabled = true
+					return env
+				},
+				Expected: []corev1.ServicePort{
+					{Name: forge.GUIPortName, Protocol: corev1.ProtocolTCP, Port: forge.GUIPortNumber, TargetPort: intstr.FromInt(forge.GUIPortNumber)},
+					{Name: forge.MetricsPortName, Protocol: corev1.ProtocolTCP, Port: forge.MetricsPortNumber, TargetPort: intstr.FromInt(forge.MetricsPortNumber)},
+				},
+			}),
```

Comment:
Wondering the difference with respect to the other cases,

Reply:
I guess this slipped while editing stuff. Can I just remove it or is there any more meaningful check you can think of? 😬

Reply:
Would remove that, as already covered by the others